core/include/traccc/edm/alt_seed.hpp

```diff
@@ -0,0 +1,50 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+#include "traccc/edm/container.hpp"
+#include "traccc/edm/spacepoint.hpp"
+
+namespace traccc {
+
+/// Seed consisting of three spacepoints, z origin and weight
+/// This differs from (non-alt) seed in the link_type definition
+struct alt_seed {
+
+    using link_type = spacepoint_collection_types::host::size_type;
+
+    link_type spB_link;
+    link_type spM_link;
+    link_type spT_link;
+
+    scalar weight;
+    scalar z_vertex;
```

Comment:
We can discuss this later, perhaps on a different PR, but this information should not really live in the seed EDM.